metadata-service/servlet/src/main/java/com/datahub/gms/servlet/Config.java

```diff
@@ -136,11 +140,19 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
       }
     }
 
-    // Serialize cached configuration
+    // Serialize cached configuration with user context
     configLock.readLock().lock();
     try {
       resp.setContentType("application/json");
-      String json = objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(cachedConfig);
+
+      // Create a copy of cached config and conditionally add user type for verbose mode
+      Map<String, Object> responseConfig = new HashMap<>(cachedConfig);
+      if (shouldIncludeVerboseInfo(req)) {
+        responseConfig.put("userType", determineUserType());
+      }
```

Comment:
I kept userType but it is enabled only if ?verbose=true parameter is provided. it is useful for smoke testing (since we can check that progressive disclosure is working)